test/runtests.jl

```diff
@@ -1,5 +1,11 @@
 using Reactant, SafeTestsets, Test
 
+# If we are in GitHub Actions, make `TMPDIR` be a local directory from which we
+# can upload artifacts at the end.
+if get(ENV, "GITHUB_ACTIONS", "false") == "true"
+    ENV["TMPDIR"] = mkpath(joinpath(@__DIR__, "..", "tmp"))
```

Comment:
Now I don't remember why I went by the route of hacking `TMPDIR` in the GB-25 repo (maybe was helpful for something else as well?  Or the var I'm going to suggest below didn't exist yet), but a more straightforward way is to use `Reactant.MLIR.IR.DUMP_MLIR_DIR`
```suggestion
    Reactant.MLIR.IR.DUMP_MLIR_DIR[] = mkpath(joinpath(@__DIR__, "..", "mlir_modules"))
```